core/spec/models/spree/promotion_action_spec.rb

```diff
@@ -3,6 +3,12 @@
 require 'rails_helper'
 
 RSpec.describe Spree::PromotionAction, type: :model do
+  describe "preferences" do
+    subject { described_class.new.preferences }
+
+    it { is_expected.to eq({}) }
```

Comment:
`be_a(Hash)` here as well?

Reply:
Nah, `{}` is fine as we know it's an empty Hash for this one. I've used `be_a(Hash)` mostly when for some reason it wasn't crystal clear what would be in there.